product_docs/docs/tpa/23/architecture-PGD-Always-ON.mdx

```diff
@@ -80,6 +86,7 @@ data centre that provides a level of redundancy, in whatever way
 this definition makes sense to your use case. For example, AWS
 regions, your own data centres, or any other designation to identify
 where your servers are hosted.
+!!!
 
 !!! Note for AWS users
```

Comment:
Probably should be 
```suggestion
!!!note Note for AWS users
```
So the Note wording is built in the deployment, otherwise it looks like this: 

![Screenshot 2024-08-22 at 09 54 13](https://github.com/user-attachments/assets/9031cf47-26d4-42be-baf5-8dc0ff527485)